src/cljs/commiteth/core.cljs

```diff
@@ -75,18 +75,22 @@
 (defn tabs [mobile?]
   (let [user (rf/subscribe [:user])
         owner-bounties (rf/subscribe [:owner-bounties])
-        current-page (rf/subscribe [:page])]
+        route (rf/subscribe [:route])]
     (fn tabs-render []
-      (let [tabs [[:bounties (str (when-not @user "Open ") "Bounties")]
+      (let [route-id (:route-id @route)
+            tabs [[:bounties (str (when-not @user "Open ") "Bounties")]
                   [:activity "Activity"]
                   (when (seq @owner-bounties)
-                    [:manage-payouts (str (when-not mobile? "Manage ") "Payouts")])
+                    [:dashboard "Dashboard"])
                   (when (:status-team-member? @user)
                     [:usage-metrics "Usage metrics"])]]
         (into [:div.ui.attached.tabular.menu.tiny]
               (for [[page caption] (remove nil? tabs)]
                 (let [props {:class (str "ui item"
-                                         (when (= @current-page page) " active"))
+                                         (if (= :dashboard page)
+                                           (when (contains? #{:dashboard :dashboard/to-confirm :dashboard/to-merge} route-id)
```

Comment:
I think you mentioned elsewhere in the codebase (in a todo or something) the value in namespacing keywords as you do here? Is there a heuristic that you are following when you feel it would be helpful?

Reply:
Admittedly it's very ad-hoc. In this case, the routes all belonged to the same user-facing "page" and so indicating that they belong together seemed helpful. I was thinking of namespacing `:dashboard` as well, i.e. `:dashboard/start`but decided against it.